OasisDEX/oasis-borrow
# call wallet with gas estimation on aave

https://app.shortcut.com/oazo-apps/story/8178/aave-gas-estimation
  
## Changes 👷‍♀️

- sends gas estimation from aave machines.
  
## How to test 🧪

- go to open/mutate aave position
- ensure gas cost on MM ("from site") is similar to what we say in the UI:

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/3783198/220895280-3dffb4f8-8f1b-468f-ba3f-2701b18a22a2.png">

Bear in mind that the txn cost could change whilst MM is being shown based on fresh block data.
